761 better getter seeters@221 fix separator div@main (#782)
insightsengineering/rtables: NEWS.md

## rtables 0.6.5.9011
### New Features
 * Removed `ref_group` reordering in column splits so not to change the order.

>  * Added `bold` argument to `as_html` to bold specified elements, and `header_sep_line` 
>    argument to print a    horizontal line under the table header in rendered HTML output.

Reviewer: ```suggestion
 * Added `bold` argument to `as_html` to bold specified elements, and `header_sep_line` 
   argument to print a horizontal line under the table header in rendered HTML output.
```

>  * Section divisors can be set also for analysis rows.

Reviewer: ```suggestion
 * Section dividers can be set for analysis rows.
```

>  * Added setter and getter for section dividers (`section_div` and `section_div<-`). They accept also

Reviewer: ```suggestion
 * Added setter and getter for section dividers (`section_div` and `section_div<-`). They also accept
```

   split section structure assignment.
 * Added `header_section_div` setters and getters for layout and table objects along with 

>    related`basic_table` parameter.

Reviewer: ```suggestion
   related `basic_table` parameter.
```

 
### Bug Fixes
 * Fixed a bug that was failing when wrapping and section dividers were used at the same time.

### Miscellaneous
 * Applied `styler` and resolved package lint. Changed default indentation from 4 spaces to 2.
 * Added Developer Guide to pkgdown site with Debugging, Split Machinery, and Tabulation sections.
 * Whitespace is not trimmed when rendering tables with `as_html`.

>  * Exported `section_div` methods have now a dedicated documentation page that is visible to users.

Reviewer: ```suggestion
 * Exported `section_div` methods now have a dedicated documentation page that is visible to users.
```

Reviewer: Updated NEWS when merging in `main`
